packages/camera/camera_platform_interface/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.11.0
 
+* Implements `setDescriptionWhileRecording` on Android. (Fixes flutter/flutter#148013)
```

Comment:
This PR only adds a flag to be set and doesn't necessarily fix the bug once released. I would change this to something like: 
```suggestion
* Adds a flag to configure a recording to be persistent across camera changes. See
  `VideoCaptureOptions.enablePersistentRecording`.
```